libs/net/template/ReportExtensions.cs

```diff
@@ -671,13 +681,35 @@ public static string[] GetLabels(IEnumerable<IGrouping<string, ContentModel>> da
     /// <param name="dataset"></param>
     /// <param name="settings"></param>
     /// <param name="sections"></param>
+    /// <param name="isDataset"></param>
     /// <returns></returns>
-    public static string GetLabel(IGrouping<string, ContentModel> dataset, API.Models.Settings.ChartSectionSettingsModel settings, Dictionary<string, ReportSectionModel> sections)
+    public static string GetLabel(IGrouping<string, ContentModel> dataset, API.Models.Settings.ChartSectionSettingsModel settings, Dictionary<string, ReportSectionModel> sections, bool isDataset)
```

Comment:
I don't like adding this new boolean argument, but it is the simplest solution without needing to rewrite a lot of code.